src/graphics/window.rs

```diff
@@ -261,27 +263,21 @@ impl Window {
     }
 
     ///Handle the available size for the window changing
-    pub(crate) fn adjust_size(&mut self, available: Vector) {
```

Comment:
I don't really get this change.. Am I accidently reverting your DPI-fix here? @ryanisaacg

Reply:
That's what it looks like, yes. Must be a weird git issue because of when this branched off.

Reply:
Yep, github says the branch is up to date 😕